### DISCUSSION TITLE
Disable normal login?

### DISCUSSION THREAD

**Discussion by poperigby** [9p4/jellyfin-plugin-sso#157] [Q&A] [answered]
I currently have the plugin setup with Authelia, but there are two options for logging in, the custom `Login with SSO` button that uses Authelia, or just logging in normally. This Authelia user has the same username as a pre-existing Jellyfin user. Can I make it so you have to go through Authelia to login?

**Comment by 9p4** [comment, marked as answer]
you could change the password for the user to something long and random and forget about it, or you could modify the database to remove the authentication provider from the user so that password auth is disabled. The first option is easier.

**Reply by MatthiasGrandl** [reply]
I am guessing option 2 isn't something this plugin could optionally enforce? I am looking into this to improve the security of my Jellyfin instance.

**Reply by MatthiasGrandl** [reply]
PS: I just tried manually removing the default authentication provider from my user, but it didn't work. Was just added back on server restart and I could login with password.

**Reply by MatthiasGrandl** [reply]
PPS: Solved this by disabling the login endpoint in my reverse proxy config. In my case for Caddy:

```
@block_login {
		method POST
		path /Users/authenticatebyname
	}
	handle @block_login {
		respond 403
	}
	```

**Reply by C8opmBM** [reply]
nice, what would be the equivalent for traefik?

**Reply by duckling-strg** [reply]
> nice, what would be the equivalent for traefik?

You could just prohibit the path itself, returning a 404: `Host(...) && !Path(/Users/authenticatebyname)`

**Reply by kriakiku** [reply]
jFYI `/Users/authenticatebyname`, `/Users/authenticatebyname/`, `/Users/Authenticatebyname`, `/Users/AuthenTicateByNaMe` — for most reverse proxies, these are treated as different paths, whereas Jellyfin considers them the same. By applying such a rule, you introduce a potential security risk.
